docs/CC2023/PAD.md

```diff
@@ -0,0 +1,51 @@
+
+# MOD2IR: High-Performance Code Generation for a Biophysically Detailed Neuronal Simulation DSL
+
+## Preliminary Artifact Description
+
+### Broad Description
+
+This artifact provides all the necessary code, scripts and results to compile the NMODL transpiler
+with the MOD2IR extension and run all benchmarks described in the manuscript. To simplify the
+evaluation process we provide along with the instructions a Dockerfile that will setup a viable
+system for the benchmarks. The driver script compiles the membrane mechanism model `hh.mod` and the
+synapse mechanism model `expsyn.mod` with various compile-time configurations and then runs the
+generated  binaries comparing their runtimes. More specifically the benchmark compares the execution
+runtime of the binaries generated via the two-step compilation process MOD-C++-binary using various
+open-source and commercial compiler frameworks with the one-step ahead-of-time and just-in-time
+processes of MOD2IR.
+MOD2IR is implemented as a code generation backend inside the NMODL Framework and it makes heavy
+use of the LLVM IR and compilation passes. Most of the relevant code of the described work can be
+found [here](https://github.com/BlueBrain/nmodl/tree/llvm/src/codegen/llvm) and
+[here](https://github.com/BlueBrain/nmodl/tree/llvm/test/benchmark). The intructions to reproduce
+the results can be found [here](https://github.com/BlueBrain/nmodl/blob/llvm/docs/CC2023/README.md).
+
+### Badge
+
+Blue Badge (results validated). We hope that using the provided Dockerfile and scripts the
```

Comment:
We can also mention red badge (artifact reusable). You can select one red and one blue.